theano/gpuarray/tests/test_type.py

```diff
@@ -79,8 +79,9 @@ def test_unpickle_gpuarray_as_numpy_ndarray_flag0():
     finally:
         config.experimental.unpickle_gpu_on_cpu = oldflag
 
-
-test_shared_options = makeSharedTester(
+# These tests are disabled because they expect the impossible
```

Comment:
The expect get_value() to return the internal type without passing return_internal_type=True.

They also expect that if you pass a ndarray and then call get_value() you get the same object.

There are a couple of options you can play with to tweak certain expectations, but those two seem hardcoded.

Overall this is not a reduction in testing since those tests were never done for gpuarray before.  This can motivate someone to actually make it work, but I don't have any more time to spend on this (and I already spent too much).